lib/pbench/test/functional/server/test_put.py

```diff
@@ -132,10 +132,8 @@ def test_archive_only(server_client: PbenchServerClient, login_user):
         # enabled by upload, and INDEX is only enabled by UNPACK: so if they're
         # not here immediately after upload, they'll never be here.
         operations = metadata["dataset.operations"]
-        assert "UNPACK" not in operations
-        assert "INDEX" not in operations
         assert operations["UPLOAD"]["state"] == "OK"
-        assert operations["BACKUP"]["state"] in ("OK", "READY", "WORKING")
+        assert "INDEX" not in operations
```

Comment:
It comes from `Sync.update`, which maintains the `Operations` table.

Reply:
It comes from `enum.Enum`'s `name` attribute...that is, since`OperationName` is derived from `enum.Enum`, `OperationName.INDEX.name` would be `"INDEX"`.